src/api-models/aplus-content-api-model/git_push.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/sh
```

Comment:
What's this file for?

I see there are many of these. We should probably delete them as the final clean up step.